public/app/plugins/datasource/loki/configuration/DerivedField.tsx

```diff
@@ -70,16 +76,50 @@ export const DerivedField = (props: Props) => {
   return (
     <div className={className} data-testid="derived-field">
       <div className="gf-form">
+        {config.featureToggles.lokiDerivedFieldsFromLabels && (
+          <Field
+            className={styles.nameMatcherField}
+            label={
+              <TooltipLabel
+                label="Type"
+                content="Derived fields can be based on a label or a regular expression base on the log message."
+              />
+            }
+          >
+            <Select
```

Comment:
Select is too small for the "Regex in log line" option
<img width="191" alt="image" src="https://github.com/grafana/grafana/assets/30407135/8c1d0b68-5c91-4227-a699-6ebdc9699558">

Reply:
Also, I think the UI would be better if we had `Name` `Type` `Regex/Label` - to make type and regex/label more connected.